libraries/AP_BattMonitor/AP_BattMonitor_Sum.cpp

```diff
@@ -103,6 +108,12 @@ AP_BattMonitor_Sum::read()
 
     update_consumed(_state, dt_us);
 
+    // Check if we want to report the minimum voltage instead of the average
```

Comment:
race condition for scripting or DDS, I think we need to create a local float state_voltage and then set that once